interface.go

```diff
@@ -63,6 +63,8 @@ type Config struct {
 	// If this field is not set, the Dial functions will return only when the connection is forward secure.
 	// Callbacks have to be thread-safe, since they might be called in separate goroutines.
 	ConnState ConnStateCallback
+	// KeepAlive defines whether keep-alive is enabled from the client to server.
```

Comment:
@lucas-clemente However, as discussed in #525, the server will never send keepalives.
So, I can remove the comment here and add it to `newSession()` that servers are not meant to send keepalives in quic-go?